test/test-server/kafka_rest_router.go

```diff
@@ -1791,8 +1791,22 @@ func handleKafkaBrokerIdConfigs(t *testing.T) http.HandlerFunc {
 }
 
 // Handler for: "/kafka/v3/clusters/{cluster_id}/broker-configs:alter"
-func handleKafkaBrokerConfigsAlter(_ *testing.T) http.HandlerFunc {
+func handleKafkaBrokerConfigsAlter(t *testing.T) http.HandlerFunc {
```

Comment:
Looks like we're still not using this
```suggestion
func handleKafkaBrokerConfigsAlter(_ *testing.T) http.HandlerFunc {
```

Reply:
Oops. At one point I thought I needed it, and I forgot to change it back when I realized I didn't.